### PR TITLE
Add option to drop extension db at un-install time or later

### PR DESCRIPTION
**Pull request by motorina0**
Requested here: https://github.com/lnbits/lnbits/issues/1709

### Summary
- by default when an extension is un-installed, all the data is still stored on the LNbits DB
- sometimes admin users might want to also remove all the data. Some reasons:
  - there was a corrupt migration file and now the extension cannot be upgraded/reinstalled
  - the existing data was created for demo/testing purposes and should be dropped
  - ...

This PR adds a `Remove Database` option at un-install time, or later.
### Tests Steps
 - see flow in screen recording below
 - use both Postgres and SQLite

### Uninstall and Drop DB
https://github.com/lnbits/lnbits/assets/2951406/8cc29d63-5698-4fba-90ee-152b5a9404a6

### Drop DB after Uninstall

https://github.com/lnbits/lnbits/assets/2951406/8d6d279d-842e-430d-abb3-de8a97b5539f

### CODE REVIEW

File: lnbits/core/static/js/service-worker.js-e [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// update cache version every time there is a new deployment
```
Comment (dni) [inline lnbits/core/static/js/service-worker.js-e:1]: lnbits/core/static/js/service-worker.js-e ? is this a error?

File: lnbits/core/static/js/extensions.js [hunk 2]
```diff
@@ -1,43 +0,0 @@
-new Vue({
```
Comment (dni) [inline lnbits/core/static/js/extensions.js:1]: was this file unnecessary?

Comment (motorina0) [reply lnbits/core/static/js/extensions.js:1]: It was a leftover before the merge of  `Extensions` (`/extensions`) and `Manage Extensions` (`/install`)

File: lnbits/core/static/js/service-worker.js-e (hunk 1 above)
Comment (motorina0) [reply lnbits/core/static/js/service-worker.js-e]: - `make bundle` generated it for me
- I have removed it

**Review by dni — APPROVED** [review]

**Review by talvasconcelos — APPROVED** [review]

**Pull request merged**